### PR TITLE
GH-564 Add sudo command

### PR DESCRIPTION
**Pull request by vLuckyyy**

### CODE REVIEW

**Review by CitralFlo — APPROVED** [review]
LGTM. I hope it was tested with all commands

**Review by igoyek — APPROVED** [review]

**Review by imDMK — APPROVED** [review]

**Review by P1otrulla — APPROVED** [review]

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/sudo/SudoCommand.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    private final Server server;
+    private final NoticeService noticeService;
+
+    @Inject
+    public SudoCommand(Server server, NoticeService noticeService) {
+        this.server = server;
+        this.noticeService = noticeService;
+    }
+
+    @Execute(name = "console")
```
Comment (Rollczi) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/sudo/SudoCommand.java:29]:
```suggestion
    @Execute(name = "-console")
```

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/sudo/SudoCommand.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @Permission("eternalcore.sudo.console")
+    @DescriptionDocs(description = "Execute command as console", arguments = "<command>")
+    void console(@Context Viewer viewer, @Arg String command) {
+        this.server.dispatchCommand(this.server.getConsoleSender(), command);
+        this.sendSudoSpy(viewer, command);
+    }
+
+    @Execute(name = "player")
```
Comment (Rollczi) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/sudo/SudoCommand.java:37]:
```suggestion
    @Execute
```

**Review by Rollczi — CHANGES_REQUESTED** [review]
We can make these commands more short e.g.
`/sudo Rollczi day`
`/sudo -console day`

**Review by Jakubk15 — APPROVED** [review]

**Review by Rollczi — APPROVED** [review]

**Pull request merged**